52 reformat dev according to pep8

HenrikhB opened the pull request:
Edited code according to PEP8 for the whole project.

Commit:
Code style changes for check app

Commit:
Code style changes for contact app

Commit:
Code style changes for daemons

Commit:
Code style changes for home, nagplugin and middleware

Commit:
Code style changes for server app

Commit:
Code style changes for service app

Commit:
Code style changes for Unit Tests

Commit:
Code style changes for registration, utils, user_profile

Commit:
Code style changes for registration app

Commit:
Minor code fixes for service views and updated coveragerc

nkoblai commented on moninag/contact/views.py:
```diff
@@ -108,19 +109,19 @@ def post(self, request):
-    def verify(request, activation_key):
-
+    def verify(request, activation_key):  # pylint: disable=no-self-argument
+        """Making contact active (active=False -> active=True)"""
```
Missing blank space after docstring.

nkoblai reviewed (approved):
@HenrikhB it works.

hutsulyk commented on moninag/home/views.py:
```diff
@@ -8,4 +10,7 @@ class IndexView(View):
     def get(self, request):
+        """
+        Method which renders main index page
+        """
```
Move to one line :slightly_smiling_face:

hutsulyk commented on moninag/nagplugin/models.py:
```diff
@@ -27,8 +29,8 @@ def get_by_id(id):
-            plugin = NagPlugin.objects.get(id=id)
-        except Exception as error:
+            plugin = NagPlugin.objects.get(id=plugin_id)
+        except Exception as error:  # pylint: disable=broad-except, unused-variable
```
```
except NagPlugin.DoesNotExist:
```
:upside_down_face:

hutsulyk commented on moninag/registration/views.py:
```diff
@@ -79,11 +90,14 @@ def register_user(request):
                 json['success'] = True
-                json['message'] = "Thank you for your time. The confirmation code has been sent to your email. In order to confirm the registration, simply click on the link given in it."
+                json[
+                    'message'] = "Thank you for your time. The confirmation code has" \
```
Why `message` on new line? :space_invader:

HenrikhB replied:
fixed 👹

Commit:
Updated .pylintrc

Modified ingored errors and warnings for pylint.

hutsulyk commented on moninag/contact/views.py:
```diff
@@ -108,19 +109,20 @@ def post(self, request):
                 return JsonResponse(json_response, status=201)
 
-        except:
+        except:  # pylint: disable=bare-except
```
`ValidationError` I guess :slightly_smiling_face:

hutsulyk commented on moninag/contact/views.py:
```diff
@@ -159,11 +161,11 @@ def put(self, request, contact_id):
             return JsonResponse(json_response, status=200)
 
-        except:
+        except:  # pylint: disable=bare-except
```
`ValidationError` :upside_down_face:

hutsulyk commented on moninag/daemons/check_launcher.py:
```diff
@@ -47,7 +47,7 @@ def inner(*args, **kwargs):
                 try:
                     return func(*args, **kwargs)
-                except Exception as error:
+                except Exception as error:  # pylint: disable=broad-except, unused-variable
```
`as error` could be removed

hutsulyk commented on moninag/registration/views.py:
```diff
@@ -79,11 +90,13 @@ def register_user(request):
                 send_activation_email(
                     DEFAULT_HOST, DEFAULT_FROM_EMAIL, user.email, activation_key)
-        except:
+        except:  # pylint: disable=bare-except
```
ValidationError :space_invader:

hutsulyk commented on moninag/registration/views.py:
```diff
@@ -111,22 +126,26 @@ def request_password_reset(request):
                 json['error'] = "No user is associated with this email address"
 
-        except:
+        except:  # pylint: disable=bare-except
```
ValidationError :horse:

Commit:
Added specific exceptions for entire project

Commit:
Updated Readme

Added project description and added info about coverage and pylint config files and launching

hutsulyk commented on moninag/registration/views.py:
```diff
@@ -147,7 +167,9 @@ def confirm_password_reset(request, uidb64=None, token=None):
             user.save()
-            json['message'] = "Password has been reset. In 5 seconds you will be redirected to the login page."
+            json[
+                'message'] = "Password has been reset. In 5 seconds you will be redirected" \
```
Move to one line :robot:

hutsulyk reviewed (approved):
Great job! :+1:
